### ISSUE TITLE
can't pass no_totals via CLI to claim search

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/lbry-sdk#2788]
```(lbry-venv) C:\Users\thoma\Documents\lbry-sdk>lbrynet claim search --no_totals
Usage:
            claim_search [<name> | --name=<name>] [--text=<text>] [--txid=<txid>] [--nout=<nout>]
                         [--claim_id=<claim_id> | --claim_ids=<claim_ids>...]
                         [--channel=<channel> |
                             [[--channel_ids=<channel_ids>...] [--not_channel_ids=<not_channel_ids>...]]]
                         [--has_channel_signature] [--valid_channel_signature | --invalid_channel_signature]
                         [--is_controlling] [--release_time=<release_time>] [--public_key_id=<public_key_id>]
                         [--timestamp=<timestamp>] [--creation_timestamp=<creation_timestamp>]
                         [--height=<height>] [--creation_height=<creation_height>]
                         [--activation_height=<activation_height>] [--expiration_height=<expiration_height>]
                         [--amount=<amount>] [--effective_amount=<effective_amount>]
                         [--support_amount=<support_amount>] [--trending_group=<trending_group>]
                         [--trending_mixed=<trending_mixed>] [--trending_local=<trending_local>]
                         [--trending_global=<trending_global]
                         [--reposted_claim_id=<reposted_claim_id>] [--reposted=<reposted>]
                         [--claim_type=<claim_type>] [--stream_types=<stream_types>...] [--media_types=<media_types>...]
                         [--fee_currency=<fee_currency>] [--fee_amount=<fee_amount>]
                         [--duration=<duration>]
                         [--any_tags=<any_tags>...] [--all_tags=<all_tags>...] [--not_tags=<not_tags>...]
                         [--any_languages=<any_languages>...] [--all_languages=<all_languages>...]
                         [--not_languages=<not_languages>...]
                         [--any_locations=<any_locations>...] [--all_locations=<all_locations>...]
                         [--not_locations=<not_locations>...]
                         [--order_by=<order_by>...] [--page=<page>] [--page_size=<page_size>]
                         [--wallet_id=<wallet_id>]
```

**Comment by patrickkeane** [comment]
Functionality is implemented by b153e4bb9fa673727a72bb2adebc957bea239f4b

There is only a small miss in the usage block.  I will link a pull request that resolves this issue.

**Comment by patrickkeane** [comment]
This would be resolved by PR #3202

**Comment by eukreign** [comment]
fixed in #3202
